drivers/osdp/Kconfig.pd

```diff
@@ -0,0 +1,217 @@
+#
+# Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config OSDP_NUM_CONNECTED_PD
+	int
+	default 1
+	help
+	  In PD mode, number of connected PDs is is always 1 and cannot
+	  be configured.
+
+config OSDP_PD_ADDRESS
+	int "PD Address"
+	default 1
+	help
+	  The 7 least significant bits represent the address of the PD to which
+	  the message is directed, or the address of the PD sending the reply.
+	  Address 0x7F is reserved as a brodcast address to which all PDs would
+	  respond.
+
+menu "PD ID Information"
```

Comment:
and `Peripheral Device ID Information` here.
For the submenus below, using PD in prompts is probably ok

Reply:
Done.